kieserver/image.yaml

```diff
@@ -142,6 +142,9 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_JBPM_CLUSTER"
+    example: "true"
+    description: "Enable the JPM clustering for Kie-Server fail-over, if less than two replica as confiured one additionl replica will be added"
```

Comment:
Can you please fix the description? reading it as a user, it is not clear to me what this env does.

Reply:
ok, using your suggestion from here https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/526